### PR TITLE
lookup elements for multiline declaration

### LINKED ISSUE #604: Lookup elements should work for multiline declaration
For 

```
pub fn get_resolved_packages<'a>(resolve: &Resolve,
                                 registry: PackageRegistry<'a>)
                                 -> PackageSet<'a> {
```

lookup element looks like

nav_info

We should try to show all parameters on the single line! 

The code is here: https://github.com/intellij-rust/intellij-rust/blob/master/src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt

### PR DESCRIPTION
**Pull request by zjhmale**
Fixes #604

### CODE REVIEW

File: src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt [hunk 1]
```diff
@@ -7,12 +7,13 @@ import org.rust.lang.core.psi.*
...
 import org.rust.lang.core.psi.util.module
 import org.rust.lang.core.psi.util.parentOfType
+import java.util.regex.Pattern
 
 fun RustNamedElement.createLookupElement(): LookupElement {
     return when (this) {
         is RustFnElement -> LookupElementBuilder.createWithIcon(this)
             .withLookupString(name ?: "")
-            .withTailText(parameters?.text ?: "()")
+            .withTailText(parameters?.text?.replace(Pattern.compile("\\s+").toRegex(), " ") ?: "()")
```
Comment (mkaput) [inline src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt]: why regexps :cry:

Comment (zjhmale) [inline src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt]: that's the best way to eliminate extra whitespaces come into my mind 😂

**Comment by matklad** [comment]
Thanks!

File: src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt (hunk 1 above)
Comment (matklad) [inline src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt]: @mkaput do you have a better solution in mind?

Comment (mkaput) [inline src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt]:
```kotlin
parameters?.text?.let { StringUtil.tokenize(it, " \t\n\r\f").joinToString(" ") } ?: "()"
```
is first thing which comes to my mind. Personally I don't like to use regexps for trivial things

Comment (matklad) [inline src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt]: I don't like to use regexps for non-trivial things :) And I have to google up `\f`.

Either solution is good.

Comment (zjhmale) [inline src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt]: cool, also googled `\f` 😂 , first time to see this special character.

**Pull request merged**